### DISCUSSION TITLE
Regex language

### DISCUSSION THREAD

**Discussion by Skippymudkip** [Chatterino/chatterino2#2964] [Q&A] [answered]
I can't find any documentation of the regex language that chatterino uses, and was wondering if anyone knew what it was. Thanks.

**Comment by KararTY** [comment]
**I assume you mean for ignores & filters.**

https://wiki.chatterino.com/Regex/
https://wiki.chatterino.com/Filters/
You can access these from Chatterino as well through the blue links in the respective tabs.


If you want to see some filters, check out these examples:
https://gist.github.com/gempir/6fef402b44e379cb2b6fe2c4f92f9741

**Comment by Felanbird** [comment, marked as answer]
As of 3b0d62f5c54bd6e2da7797a981daf898624d26ed we use https://doc.qt.io/qt-5/qregularexpression.html

![image](https://user-images.githubusercontent.com/41973452/124993315-35b4b300-e012-11eb-910c-2955b669685d.png)
